addons/point_of_sale/static/tests/tours/product_screen_tour.js

```diff
@@ -269,11 +269,18 @@ registry.category("web_tour.tours").add("DecimalCommaOrderlinePrice", {
         ].flat(),
 });
 
-registry.category("web_tour.tours").add("CheckProductInformation", {
+registry.category("web_tour.tours").add("MiscTestPosAdmin", {
     test: true,
     steps: () =>
         [
             Dialog.confirm("Open session"),
+
+            // Check that the product form is shown.
+            Chrome.clickMenuButton(),
+            Chrome.clickMenuDropdownOption("Create Product"),
+            Dialog.cancel(),
```

Comment:
I would prefer a step looking if the product popup is displayed than a step cancelling the dialog. This is because, if there is an error opening an alert dialog, this test will still pass where we do not want it to pass so maybe just add a step checkProductDialog or something like that.

Reply:
I added a check on the title of the dialog.